test/wsdl/recursive/A.xsd

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xs:schema targetNamespace="http://www.Dummy.com/Common/Types"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="http://www.Dummy.com/Common/Types"
+  >
+  <xs:import namespace="http://www.Dummy.com/Name/Types" schemaLocation="B.xsd" />
```

Comment:
This should mimic 


    file.wsdl
        xs:import namespace="A" schemaLocation: A.xsd
    A.xsd
        xs:import namespace="B" schemaLocation: B.xsd
    B.xsd
        xs:import namespace="A" schemaLocation: A.xsd